sonic_data_client/db_client.go

```diff
@@ -496,6 +506,12 @@ func makeJSON_redis(msi *map[string]interface{}, key *string, op *string, mfv ma
 		for f, v := range mfv {
 			(*msi)[f] = v
 		}
+		// Debug log
+		//log.V(5).Infof("msi: %v\n", msi)
+		//log.V(5).Infof("key: %v\n", key)
+		//log.V(5).Infof("op: %v\n", op)
```

Comment:
Remove these

Reply:
Removed.

Reply:
They don't appear to have been removed.

Reply:
Ah these were in multiple files. I removed them from `virtual_database_client` but this one was under `sonic_data_client`.

Reply:
Please remove if you have not. Thanks!